docs/AvroDynamicPartitionedDataset-batchsink.md

```diff
@@ -0,0 +1,69 @@
+# Avro Dynamic Partitioned Dataset Batch Sink
+
+
+Description
+-----------
+Sink for a ``PartitionedDataset`` that writes data in Avro format
+and leverages one or more record field values for creating partitions.
+All data for the run will be written to a partition based on the
+specified fields and their value.
+
+
+Use Case
+--------
+This sink is used whenever you want to write to a ``PartitionedFileSet`` in Avro format
+using a value from the record as a partition. For example, you might want to load historical
+data from a database and partition the dataset on the original creation date of the data.
+
+
+Properties
+----------
+| Configuration | Required | Default | Description |
+| :------------ | :------: | :------ | :---------- |
+| **name** | **Y** | None| Name of the ``PartitionedFileSet`` to which records are written. If it doesn't exist, it will be created. |
```

Comment:
Can you make these names the same as what shows up in the UI for the plugin? Base Path instead of basePath for example?